test/functional/api/vim_spec.lua

```diff
@@ -2449,7 +2449,8 @@ describe('API', function()
     }
 
     it('returns {} for invalid channel', function()
-      eq({}, api.nvim_get_chan_info(0))
+      -- 0 should return current channel
+      eq(testinfo, api.nvim_get_chan_info(0))
```

Comment:
I think this should be moved to the `stream=stdio channel` block, before both `eq(testinfo, api.nvim_get_chan_info(1))` assertions.